vMenuServer/HttpRequests.cs

```diff
@@ -0,0 +1,126 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using CitizenFX.Core;
+using CitizenFX.Core.Native;
+using Newtonsoft.Json;
+
+namespace vMenuServer
+{
+	// FiveMHttpRequests class is made by Blade: https://gist.github.com/bladecoding/368f6a57108696c350fa7408f387a482
+	// Some stuff was added for discord webhook requests
```

Comment:
The indentation is all screwed up in this file. Also I'd separate Discord utilities from the actual web request code.